### DISCUSSION TITLE
Deckhouse registry license

### DISCUSSION THREAD

**Discussion by Agalin** [deckhouse/k8s-image-availability-exporter#190] [General]
Hi! I know some people may find this question silly but what's the access policy / license of the Deckhouse container registry?

Project itself is licensed as Apache but that doesn't mean we can freely use prebuilt images, in some cases (e.g. CrunchyData Postgres Operator) building from source is necessary as registry itself has non-free access policy.

So how is it with Deckhouse? Can we use it in production? Or maybe there is a Docker Hub / Quay.io / GHCR / whatever mirror of the image?

**Comment by nabokihms** [comment]
For production usage, it is a common practice to store images in your company's registry (or using a proxy registry). You do not want whichever public registry be the source of your production outage.

**Reply by Agalin** [reply]
But that doesn't answer the question. I can only use a caching proxy if Deckhouse registry usage is allowed.
